.github/release-drafter.yml

```
@@ -0,0 +1,9 @@
+_extends: jellyfin-meta-plugins
+
+template: |
+  <!-- Optional: add a release summary here -->
+  [Plugin build can be downloaded here](https://repo.jellyfin.org/releases/plugin/jellyfin-plugin-trakt/jellyfin-plugin-trakt_$NEXT_MAJOR_VERSION.0.0.0.zip).
```

```suggestion
  [Plugin build can be downloaded here](https://repo.jellyfin.org/releases/plugin/trakt/trakt_$NEXT_MAJOR_VERSION.0.0.0.zip).
```
jellyfin-plugin-* is the old path, they go in the dir without now (slugified plugin name)
https://repo.jellyfin.org/releases/plugin/trakt/trakt_10.0.0.0.zip
https://github.com/jellyfin/jellyfin-plugin-trakt/blob/a07a2897d513c7c7c728917bc800ba3783f0441a/build.yaml#L2

---

aaah I see, was wondering why I only saw such a select few in the official repo :sweat_smile: (I should not shotgun open PRs at this time of day, 1am)
Thank you for the heads up, this will have happened to a few PRs then, sorry